mangaki/mangaki/utils/common.py

```diff
@@ -1,24 +1,37 @@
 from django.conf import settings
 from mangaki.utils.chrono import Chrono
+from sklearn.metrics import mean_squared_error
 import pickle
 import os.path
 
 
+VERBOSE = False
```

Comment:
Nit / relatively important to me: it should be a Django's config parameter, let's say `VERBOSE = settings.RECO_ALGORITHM_VERBOSE`.